Robot/InfiniteRecharge/src/main/java/frc/robot/commands/TestMotorCommand_CAN1618.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.commands;
+
+import frc.robot.RobotContainer;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import java.util.logging.Logger;
+
+/**
+ * An example command that uses an example subsystem.
+ */
+public class TestMotorCommand_CAN1618 extends CommandBase {
+  @SuppressWarnings({ "PMD.UnusedPrivateField", "PMD.SingularField" })
+
+  private final Logger logger = Logger.getLogger(this.getClass().getName());
+
+  public TestMotorCommand_CAN1618() {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(RobotContainer.getInstance().shooter);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    logger.info("got to Shooter motor Activate");
+    RobotContainer.getInstance().shooter.LowGoalSpin();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.